crates/router/src/connector/nuvei/transformers.rs

```diff
@@ -73,6 +73,7 @@ pub struct NuveiPaymentsRequest {
     pub transaction_type: TransactionType,
     pub is_rebilling: Option<String>,
     pub payment_option: PaymentOption,
+    pub device_details: Option<DeviceDetails>,
```

Comment:
device_details shouldn't be an optional fields. Since ipAddress is mandatory as per the API docs of Nuvei - https://docs.nuvei.com/api/main/indexMain_v1_0.html?json#payment

Same applicable for billing_address and its fields

Reply:
@ArjunKarthik By default It is mandatory, but the fields can be mark it as optional by taking to the Nuvei gateway, so we should treat this an optional and let the gateway decline the transaction or throw the validation error.

cc @cb-alfredjoseph

Reply:
As discussed this should be declined by Gateway instead of mandated at Hyperswitch side. Resolving this.